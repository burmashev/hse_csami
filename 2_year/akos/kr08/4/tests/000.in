std_accumulate((void*)(&result), &{1, 2, 3, 4}, sizeof(int), 4, &int_zero, summator)
