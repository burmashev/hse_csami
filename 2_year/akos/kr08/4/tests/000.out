printf("%d\n", *result);
10
